de.fhdo.lemma.intermediate.transformations/transformation/IntermediateOperationModelTransformation.atl

```diff
@@ -469,44 +477,44 @@ rule IntermediateOperationEndpoint(sourceEndpoint : Operation!BasicEndpoint,
         );
     }
     to
-        t : Intermediate!IntermediateOperationEndpoint(
+        endPoint : Intermediate!IntermediateOperationEndpoint(
```

Comment:
`endPoint` should be captialized `endpoint` to fit the intermediate metamodel capitalization.